From be443adb0f0f22025dbbfba15189a186dbc7cdc3 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Wed, 19 Dec 2018 15:28:31 +0100
Subject: [PATCH] systemd: Remove WantedBy=sockets.service in [Install]

.socket systemd units already automatically get a
Before=sockets.service, and the udev rule we have will ensure that when
the appropriate virtio device is there, the socket unit will be started,
so this WantedBy dependency is not needed, and is in part responsible
for these warnings:

[ INFO ] dev-virtio\x2dports-com.redhat.spice.0.device is not active.
[DEPEND] Dependency failed for Activation socket for spice guest agent daemon.

https://bugzilla.redhat.com/show_bug.cgi?id=1545212

Signed-off-by: Christophe Fergeau <cfergeau@redhat.com>
(cherry picked from commit 56d50d21223e6d9aec4a68400d8e36c5953ba3cc)
---
 data/spice-vdagentd.socket | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/data/spice-vdagentd.socket b/data/spice-vdagentd.socket
index f14e662..adace91 100644
--- a/data/spice-vdagentd.socket
+++ b/data/spice-vdagentd.socket
@@ -3,6 +3,3 @@ Description=Activation socket for spice guest agent daemon
 
 [Socket]
 ListenStream=/var/run/spice-vdagentd/spice-vdagent-sock
-
-[Install]
-WantedBy=sockets.target
-- 
2.20.1

